imports/services/isEditedService.js

```diff
@@ -0,0 +1,279 @@
+import { Meteor } from 'meteor/meteor';
+import '/imports/helpers/promisedMethods';
+
+import { MinutesSchema } from '../collections/minutes.schema';
+import { Minutes } from '../minutes';
+import { MeetingSeriesSchema } from '../collections/meetingseries.schema';
+import { MeetingSeries } from '../meetingseries';
+import {Topic} from '../topic';
+
+function setIsEditedMS(msId) {
+    let ms = new MeetingSeries(msId);
+
+    ms.isEditedBy = Meteor.userId();
+    ms.isEditedDate = new Date();
+
+    ms.save();
+}
+
+function removeIsEditedMS(msId, any) {
+    let unset = false;
+    let ms = new MeetingSeries(msId);
+
+    if (any === true) {
+        unset = true;
+    }
+    else {
+        if (ms.isEditedBy === Meteor.userId()) {
+            unset = true;
+        }
+    }
+
+    if (unset === true) {
+        ms.isEditedBy = null;
+        ms.isEditedDate = null;
+        ms.save();
+    }
+}
+
+function removeIsEditedMin(minuteId, any) {
+    let minute = new Minutes(minuteId);
+    for (let topic of minute.topics) {
+        if (any === true) {
+            topic.isEditedBy = null;
+            topic.isEditedDate = null;
+        }
+        else {
+            if (topic.isEditedBy === Meteor.userId()) {
+                topic.isEditedBy = null;
+                topic.isEditedDate = null;
+            }
+        }
+        for (let infoItem of topic.infoItems) {
+            if (any === true)
+            {
+                infoItem.isEditedBy = null;
+                infoItem.isEditedDate = null;
+            }
+            else {
+                if(infoItem.isEditedBy === Meteor.userId()) {
+                    infoItem.isEditedBy = null;
+                    infoItem.isEditedDate = null;
+                }
+            }
+            for (let detail of infoItem.details) {
+                if (any === true) {
+                    detail.isEditedBy = null;
+                    detail.isEditedDate = null;
+                }
+                else {
+                    if (detail.isEditedBy === Meteor.userId()) {
+                        detail.isEditedBy = null;
+                        detail.isEditedDate = null;
+                    }
+                }
+            }
+        }
+    }
+    minute.save();
+}
+
+function setIsEditedTop(minutesId, topicId) {
```

Comment:
Please write full names: `setIsEditedTopic` is much clearer.

Reply:
The same applies to other functions here, e.g. `setIsEditedII` & `setIsEditedMin`. Data storage is cheap nowadays, we don't need to save every byte.